.github/workflows/release-pypi.yml

```diff
@@ -66,6 +66,3 @@ jobs:
       - name: Publish package to PyPI
         if: startsWith(github.event.ref, 'refs/tags')
         uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
I think this should work, nonetheless just invoking `uv publish` also suffices, without having to use any extra workflow, maybe we just simplify further?